apps/builder/app/shared/db/user-plan-features.server.ts

```diff
@@ -71,6 +72,18 @@ export const getUserPlanFeatures = async (
     };
   }
 
+  if (env.USER_PLAN === "pro") {
+    return {
```

Comment:
How are we going to maintain integrity of these flags? Shouldn't there be a type or someting? I know we have this list in multiple places now

Reply:
please don't merge it without making sure we can keep that list consistent everywhere

Reply:
env can be parsed with zod for now we don't do that

Reply:
we need to make sure this list is consistent with everything else ...

Reply:
otehrwise this is high risk for tech debt

Reply:
This variable has no effect on production, if it's set to any non pro value nothing bad would happen.